components/infra-proxy-service/storage/postgres/migrations.go

```diff
@@ -145,6 +144,21 @@ func (p *postgres) insertMigration(ctx context.Context, migrationId, serverId, m
 	return m, nil
 }
 
+//GetActiveMigration gets the Migration ID and Migration Status for a server id
+func (p *postgres) GetActiveMigration(ctx context.Context, serverId string) (storage.ActiveMigration, error) {
+	var m storage.ActiveMigration
+	query := "select m.id,t.type from migration m join migration_type t on m.type_id=t.id and t.id < 5000 and m.server_id=$1 order by updated_timestamp desc FETCH FIRST ROW ONLY"
```

Comment:
Are we not dealing with the migration_status here as In Progress,completed or failed?

Reply:
no, as in the migration _type table the statuses are clearly mentioned

Reply:
okay

Reply:
I am assuming that if the migration is not completed or failed then we should be in any of the phases and in progress